code/renderers/solid/template/cli/js/Button.stories.jsx

```diff
@@ -0,0 +1,13 @@
+import { Button } from './Button';
+
+// More on how to set up stories at: https://storybook.js.org/docs/7.0/html/writing-stories/introduction
+export default {
+  title: 'Example/Button',
+  tags: ['autodocs'],
+  render: (args) => <Button {...args}/>,
+};
+
+// More on writing stories with args: https://storybook.js.org/docs/7.0/html/writing-stories/args
+export const Primary = {
+  args: {},
```

Comment:
There should probably be some content on the button to not confuse people.

```suggestion
  args: { children: 'Click Me' },
```